src/components/menuFilter/demoBasicUsage/index.html

```diff
@@ -0,0 +1,26 @@
+<div ng-controller="menuFilterController as ctrl" class="md-padding" ng-cloak>
+  <div>
+    <h1 class="md-title">Pick a drink below</h1>
+    <div layout="row">
+      <md-input-container>
+        <label>Drinks</label>
+        <md-select ng-model="ctrl.selectedDrinks" multiple
+                   md-on-close="ctrl.clearSearch()"
+                   data-md-container-class="menuFilterdemoBasicUsage">
+          <md-select-header class="demo-select-header">
+            <md-menu-filter items="ctrl.drinks"
```

Comment:
@ThomasBurleson 

The hope for this directive is essentially to have a directive that a user can drop in and give a list of items (an array of strings) and that will give them a list of matching items (based on what the user types in the search input).

In essence we wanted a directive users could just drop anywhere that takes a list of strings and gives them an input box to filter those items.

It will be up to the developer on how they want to display those filtered items.

I agree the name could be better, but I am still trying to think of a better description for what is going on here, any suggestions off the top of your head?